rcmgr.go

```diff
@@ -89,10 +95,12 @@ var _ network.PeerScope = (*peerScope)(nil)
 type connectionScope struct {
 	*resourceScope
 
-	dir   network.Direction
-	usefd bool
-	rcmgr *resourceManager
-	peer  *peerScope
+	dir           network.Direction
+	usefd         bool
+	isAllowlisted bool
```

Comment:
Nit: `isAllowListed`

Reply:
I meant to call this `allowlist` or `Allowlist`. Purposely not capitalizing the l. I realize I should change `newAllowList` to `newAllowlist` to be consistent.